template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/content/ClassLibrary-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "icon": "ide/ClassLibraryCS.ico",
+  "order": 20,
+  "tags": [
+    {
+      "type": "projectType",
+      "add": [],
+      "remove": [ "Common" ]
+    }
+  ],
```

Comment:
@baronfel do we need to keep it? would it be better to remove this tag from `template.json`?

Reply:
+1 If we don't need it in the tag list:

Class Library                                 classlib             [C#],F#,VB  **Common**/Library

Reply:
Tags are used in filtering on the CLI and VS, right?  What does "Common" mean here, semantically? If anything it feels like something that's host-specific - move templates used recently to the top of the list, etc.

Reply:
in CLI it is possible to filter by tags when doing `list` or `search` using `--tag` option.
Those templates have `Common` tag:
```
PS C:\Users\vshubina> dotnet new list --tag Common
These templates matched your input: --tag='Common'

Template Name                              Short Name           Language    Tags
-----------------------------------------  -------------------  ----------  -----------------
Class Library                              classlib             [C#],F#,VB  Common/Library
Console App                                console              [C#],F#,VB  Common/Console
Web api application using Microsoft Graph  graphwebapi          [C#]        Common/Web
Windows Forms App                          winforms             [C#],VB     Common/WinForms
Windows Forms Class Library                winformslib          [C#],VB     Common/WinForms
Windows Forms Control Library              winformscontrollib   [C#],VB     Common/WinForms
Worker Service                             worker               [C#],F#     Common/Worker/Web
WPF Application                            wpf                  [C#],VB     Common/WPF
WPF Class library                          wpflib               [C#],VB     Common/WPF
WPF Custom Control Library                 wpfcustomcontrollib  [C#],VB     Common/WPF
WPF User Control Library                   wpfusercontrollib    [C#],VB     Common/WPF
```

If you do `dotnet new search --tag Common --columns tags` there are lots of hits for many things.